R/with.R

```diff
@@ -62,7 +62,7 @@ with_defaults <- function(..., default = default_linters) {
     bad_nms <- setdiff(nms[to_null], names(default))
     is_are <- if (length(bad_nms) > 1L) "are" else "is"
     warning(
-      "Trying to remove ", glue::glue_collapse(sQuote(bad_nms), sep = ", ", last = " and "),
+      "Trying to remove ", glue::glue_collapse(quote_wrap(bad_nms, "'"), sep = ", ", last = " and "),
```

Comment:
I deliberately chose `sQuote()` here to allow use of fancy quotes.
Testing for the warning could still be done by using `rex::rex("...", anything, "quoted", anything)` if we really want to test the whole message.

Reply:
I see. The issue was the fancy quotes are multi-byte, so regex `.` doesn't capture them. I'll revert.